cli/azd/pkg/infra/provisioning/bicep/bicep_provider_test.go

```diff
@@ -1032,17 +1032,18 @@ func TestUserDefinedTypes(t *testing.T) {
 		},
 		objectParam.Properties)
 	require.NotNil(t, objectParam.AdditionalProperties)
```

Comment:
Just to note, since previously AdditionalProperties was not typed as a `*` value, I think this was always returning true, since the default value would be the zero value which was non nil.

In my change, I made this a pointer type.